student-info-system/sis/views.py

```diff
@@ -0,0 +1,7 @@
+from django.shortcuts import render
+
+from django.http import HttpResponse
+
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the polls index.")
```

Comment:
Not a big deal but if you do end up changing anything before we merge this in, please change this to student info system index or something relevant.